modules/seqexec/server/src/main/scala/seqexec/server/package.scala

```diff
@@ -125,15 +129,37 @@ package object server {
       ab.fold(a => Validated.Invalid(NonEmptyList.of(a)), b => Validated.Valid(b))
   }
 
-  implicit class ExecutionQueueOps(q: ExecutionQueue) {
-    def status(st: EngineState): SequenceState = {
-      val statuses: List[SequenceState] = q.map(st.executionState.sequences.get(_).map(_.status).getOrElse(SequenceState.Idle))
+  // This assumes that there is only one instance of e in l
+  private def moveElement[T](l: List[T], e: T, d: Int)(implicit eq: Eq[T]): List[T] = {
```

Comment:
What does this method do?

Reply:
It moves the element `e` `d` positions up or down inside the list `l`.

Reply:
Sorry, I mean, can you add a comment explaining what it does?